pkg/cmd/cli/key_test.go

```diff
@@ -0,0 +1,93 @@
+package cli
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"testing"
+
+	"optable-pair-cli/pkg/keys"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestKeyCreate(t *testing.T) {
+	t.Parallel()
+
+	tmpDir := os.TempDir()
+	keyConfigFile := tmpDir + "/test_config.json"
```

Comment:
Maybe use os.TempFile?

Reply:
Eventhough /tmp files are automatically deleted once the computer is restarted, you should still delete the "extraneous" files that you have created after the test is done

os.TempFile(...)
defer os.RemoveFile() // or whatever it is

Reply:
CreateTemp creates a new temporary file in the directory dir, opens the file for reading and writing, and returns the resulting file. I prefer to test if its created.

Reply:
Ah, ok

You still need to delete the temp dir after it is created, though